src/Entity/Controller/SlotListBuilder.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Drupal\acquia_lift\Entity\Controller;
+
+use Drupal\acquia_lift\AcquiaLiftException;
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+use Drupal\Core\Config\Entity\ConfigEntityListBuilder;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use GuzzleHttp\Exception\RequestException;
+
+/**
+ * Builds a listing of slot entities.
+ */
+class SlotListBuilder extends ConfigEntityListBuilder {
+
+  /**
+   * The Lift API Helper.
+   *
+   * @var \Acquia\LiftClient\Lift
+   */
+  protected $liftClient;
+
+  /**
+   * Constructs an IndexListBuilder object.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeInterface $entity_type
+   *   The entity type definition.
+   * @param \Drupal\Core\Entity\EntityStorageInterface $storage
+   *   The entity storage class.
+   * @param  \Drupal\acquia_lift\Service\Helper\LiftAPIHelper
+   *   The Lift API Helper.
+   */
+  public function __construct(EntityTypeInterface $entity_type, EntityStorageInterface $storage) {
+    parent::__construct($entity_type, $storage);
+    try {
+      /** @var \Drupal\acquia_lift\Service\Helper\LiftAPIHelper $liftHelper */
+      $liftHelper = \Drupal::getContainer()
+        ->get('acquia_lift.service.helper.lift_api_helper');
```

Comment:
That's the problem, the helper requires the passwords to be set, and Drupal goes crazy if it cannot instantiate the service.  If we want to inject it, it means that the connection won't be ready until you call a function to "loadCredentials" or something similar before you can start and use the service. To me, loading the service and having it throw errors if not all the credentials are there is a better DX for custom use compared to enforcing another function to happen first before we can use it. Thinking this through, I could make every function call the loadCredentials phase before use so this becomes a possibility. I'll see what I can do :)